sdk/python/kfp/compiler/compiler.py

```diff
@@ -1091,10 +1091,20 @@ def _validate_workflow(workflow: dict):
   if argo_path:
     has_working_argo_lint = False
     try:
-      has_working_argo_lint = _run_argo_lint('')
+      has_working_argo_lint = _run_argo_lint("""
+      apiVersion: argoproj.io/v1alpha1 
+      kind: Workflow 
+      metadata: 
+        generateName: hello-world- 
+      spec: 
+        entrypoint: whalesay 
+        templates: 
+        - name: whalesay 
+          container: 
+            image: docker/whalesay:latest""")
     except:
-      warnings.warn("Cannot validate the compiled workflow. Found the argo program in PATH, but it's not usable. argo v2.4.3 should work.")
-
+      warnings.warn("Cannot validate the compiled workflow. Found the argo program in PATH, but it's not usable. argo v2.12.10 should work.")
```

Comment:
For this PR, when people see this error message, I think we should let me keep using v2.4.3.

However, if they do have a kubeconfig, they can simply use later versions of argo CLI.

Reply:
Sounds good to me, I will check the work required in argo to make it work locally without a kubeconfig as well.